docs/csharp/language-reference/keywords/snippets/filescopednamespace.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace SampleFileScopedNamespace;
+
+class SampleClass { }
+
+interface ISampleInterface { }
+
```

Comment:
What happens if I inject another `namespace` here? Does that change the remaining type defs? Do you mention this limitation or ability in anywhere?

Reply:
I added an example to clarify those restrictions. (It's not allowed).